src/pages-and-resources/discussions/app-config-form/apps/shared/DiscussionRestriction.jsx

```diff
@@ -0,0 +1,125 @@
+        {intl.formatMessage(messages.discussionRestrictionLabel)}
+      </h5>
+      <Tabs
+        defaultActiveKey="off"
+        id="uncontrolled-pills-tab-example"
+        variant="button-group"
+        className="mb-3 w-100 d-flex flex-row height-36"
+      >
+        <Tab eventKey="off" title="Off" tabClassName="flex-grow-1 text-center rounded-0 font-size-14 line-height-20">
+          <div className="small text-muted font-size-14 height-24 mb-4">
+            {intl.formatMessage(messages.discussionRestrictionHelp)}
+          </div>
+        </Tab>
+        <Tab eventKey="on" title="On" tabClassName="flex-grow-1 text-center font-size-14 line-height-20">
+          <div className="small mb-4 text-muted font-size-14 height-24">
+            {intl.formatMessage(messages.discussionRestrictionHelp)}
+          </div>
+          <ConfirmationPopup
+            label={intl.formatMessage(messages.enableRestrictedDatesConfirmationLabel)}
+            bodyText={intl.formatMessage(messages.enableRestrictedDatesConfirmationHelp)}
+            onConfirm={() => {}}
+            confirmLabel={intl.formatMessage(messages.ok)}
+            cancelLabel={intl.formatMessage(messages.cancelButton)}
+            confirmVariant="plain"
+            confirmButtonClass="bg-primary-500 text-white rounded-0 action-btn"
+            cancelButtonClass="rounded-0 action-btn"
+            confirmBodyClass="card-body-section"
+          />
+        </Tab>
+        <Tab eventKey="scheduled" title="Scheduled" tabClassName="flex-grow-1 text-center font-size-14 line-height-20">
+          <div className="small mb-3 text-muted font-size-14 height-24">
+            {intl.formatMessage(messages.discussionRestrictionDatesHelp)}
+          </div>
+          <div>
+            <FieldArray
+              name="restrictedDates"
+              render={({ push, remove }) => (
+                <div>
+                  {restrictedDates.map((restrictedDate, index) => (
+                    <DiscussionRestrictionItem
+                      fieldNameCommonBase={`restrictedDates.${index}`}
+                      restrictedDate={restrictedDate}
+                      key={`date-${restrictedDate.id}`}
+                      id={restrictedDate.id}
+                      onDelete={() => remove(index)}
+                      onClose={() => handleOnClose(index)}
+                      hasError={Boolean(errors?.restrictedDates?.[index])}
+                    />
+              ))}
+                  <div className="mb-4 height-36">
+                    <Button
+                      onClick={() => onAddNewItem(push)}
+                      variant="link"
+                      iconBefore={Add}
+                      className="text-primary-500 p-0"
+                      style={{ height: 28 }}
+                    >
+                      {intl.formatMessage(messages.addRestrictedDatesButton)}
+                    </Button>
+                  </div>
+                </div>
```

Comment:
replace tabs structure to button group as tabs structure in changing the UI in mobile view